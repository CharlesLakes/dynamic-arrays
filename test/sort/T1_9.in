10
43 8 12 51 48 38 40 47 48 39