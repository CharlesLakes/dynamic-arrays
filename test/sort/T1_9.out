8 12 38 39 40 43 47 48 48 51